core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -47,7 +47,7 @@
 			'keys' => array
 			(
 				'id' => 'primary',
-				'pid,ptable,invisible,sorting' => 'index'
+				'pid,invisible,ptable,start,stop' => 'index'
```

Comment:
Why is `invisible` before `ptable` here? `pid_ptable` is the defacto FK.

Reply:
Fixed in 42796f74563a417e671edabff14ec78e91249ec6.